lib/random.gd

```diff
@@ -237,6 +259,70 @@ fi;
 DeclareOperation( "RandomSource", [IsOperation] );
 DeclareOperation( "RandomSource", [IsOperation, IsObject] );
 
+
+##############################################################################
+##
+##  <#GAPDoc Label="RandomSource_develop">
+##  <Subsection Label="Implementing new kinds of random sources">
+##  <Heading>Implementing new kinds of random sources</Heading>
+##
+##  If one wants to implement a new kind of random sources then
+##  the first step is the declaration of a new category <C>C</C>, say,
+##  that implies <Ref Filt="IsRandomSource"/>, analogous to the categories
+##  listed in Section <Ref Subsect="Kinds of Random Sources"/>.
```

Comment:
Perhaps we should be explicit and mention this is usually done by calling `DeclareCategory("C", IsRandomSource);`

Reply:
Good idea.